js/header.js

```diff
@@ -28,25 +28,27 @@ addConfigOptions = function (category, options) {
 	else CONFIG_OPTIONS[category] = Object.assign(CONFIG_OPTIONS[category], options);
 };
 
-//OBJECT_DEFINE_PROPERTY = Object.defineProperty; // FIXME(165) re-enable when we have a better solution
+OBJECT_DEFINE_PROPERTY = Object.defineProperty;
+OBJECT_DEFINED_COUNT = 0;
```

Comment:
can you explain what this is doing/why it is necessary? i.e., have you tested without this counter/reset-to-original-function?

Reply:
Without resetting to original, it crashed somewhere later during roll20 code execution because of (as we now know, thanks to you) missing `return` statement. I didn't figure out the reason, but resetting to original function immediately after altering the account permissions also did the trick.